### DISCUSSION TITLE
Explanations of variables in example codes

### DISCUSSION THREAD

**Discussion by JianboLong** [mfem/mfem#3866] [Q&A]
Hello, 

I am looking for the meaning of a particular variable, kappa,

`kappa = freq * M_PI;`

that is related to the ordinary frequency in solving Maxwells equations in the example codes (ex3 and ex31, for instances). Can anyone explain the meaning of it ? More genearally, I find the exact meanings of some variables in the examples are not well commented in the given examples. Does anyone know where we can get help regarding this matter ? Any detailed documentation online ? 

Thanks !

**Comment by tzanio** [comment]
ping: @mlstowell

**Comment by v-dobrev** [comment]
In example 3, `kappa` is just an auxiliary variable used in the definition of the manufactured solution (function `E_exact`) used by the example. The `freq` variable controls the "frequency" of the manufactured solution in each spatial direction -- when `freq = 1` (default) then the solution has one full sine wave over a distance of 2.

In example 31, I think it used similarly -- see the function `E_exact` for specifics.
